src/Card/Card.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Login from '../Login/Login.js';
+import ShowPage from '../ShowPage/ShowPage';
+import { Route, Switch, Link, NavLink } from 'react-router-dom';
```

Comment:
Let's spend some time cleaning up the code before we get started on our next work session. Some of these files have unnecessary imports and if we nuked them, it would clear out some of the extraneous warnings and errors.

Reply:
I recommend that y'all make these changes to the PR before you merge it :D Makes your life much easier and reflects true "real world" workflow, where a PR is pushed up, commented on, updated, commented on some more, updated some more, finalized, and then merged!